assemblies/pentaho-cde-renderer/feature/src/main/feature/feature.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
+
+  <feature name="foundry-cde-renderer" version="${project.version}">
+    <feature version="${pentaho.version}">pentaho-requirejs-osgi-manager</feature>
+    <feature version="${pentaho.version}">pentaho-webpackage</feature>
+  </feature>
+
+  <feature name="pentaho-requirejs-osgi-manager" version="${pentaho.version}">
```

Comment:
Is this temporarily hardcoded here because we don't have it as a published artifact?

Reply:
Use the proper version property name: https://github.com/pentaho/build-resources/blob/8.1.0.0/version.properties#L101